include/LLGL/Backend/OpenGL/Linux/LinuxNativeHandle.h

```diff
@@ -9,15 +9,24 @@
 #define LLGL_OPENGL_LINUX_NATIVE_HANDLE_H
 
 
+#include <LLGL/Deprecated.h>
 #include <GL/glx.h>
 
+#if LLGL_EXPOSE_WAYLAND || LLGL_LINUX_ENABLE_WAYLAND
+    #include <EGL/egl.h>
+#endif
+
 
 namespace LLGL
 {
 
 namespace OpenGL
 {
 
+enum class RenderSystemNativeHandleType : char {
```

Comment:
Minor nitpick: can we name this `RenderSystemNativeType` instead? This would match the pattern compared to `LLGL::Direct3D12::ResourceNativeHandle`/ResourceNativeType` and shorten this long identifier a little bit (as opposed to `LLGL::OpenGL::RenderSystemNativeHandleType::EGL`).